WooCommerce/Classes/ViewRelated/Products/ProductsListViewModel.swift

```diff
@@ -12,20 +12,15 @@ class ProductListViewModel {
     let siteID: Int64
     private let stores: StoresManager
     private let userDefaults: UserDefaults
```

Comment:
We can remove the `userDefaults` as we no longer use it.

Reply:
Thanks! Removed in https://github.com/woocommerce/woocommerce-ios/pull/10960/commits/33bd112d7e4a7abb2a869a95f57c6d622465ceb8